lib/Main.js

```diff
@@ -381,6 +397,7 @@ Main.prototype.listRoomsFor = function(intent, state) {
 };
 
 Main.prototype.onMatrixEvent = function(ev) {
+    console.log("Received matrix event:", JSON.stringify(ev));
```

Comment:
I think this will be too much verbose. We should use some debug mode and not always enable it.